### ISSUE TITLE
[Feature Request]: Collapsable sections in quest descriptions

### ISSUE THREAD

**Issue by aaronhowser1** [FTBTeam/FTB-Mods-Issues#526]
### Mod

FTB Quests

### Description

<details>
  <summary>Example (click this)</summary>
  
 Like this
  
</details>

Basically so I can have a massive amount of text, but not require the entire thing be visible at once. Would be nice for like, structure build instructions.

**Comment by aaronhowser1** [comment]
In efforts of saving vertical space, I've abused the new hover text function.

![](https://cdn.discordapp.com/attachments/166630061217153024/1073000320449380492/image.png)

This is a horrible pain to write, however. This is what that paragraph looks like:

["",{"translate":"university.dank.desc4"},{"translate":"university.dank.normal","color":"light_purple","hoverEvent":{"action":"show_text","contents":[{"translate":"university.dank.normaltext"}]}},", ",{"translate":"university.dank.all","color":"light_purple","hoverEvent":{"action":"show_text","contents":[{"translate":"university.dank.alltext"}]}},", ",{"translate":"university.dank.filtered","color":"light_purple","hoverEvent":{"action":"show_text","contents":[{"translate":"university.dank.filtered"}]}},", ",{"translate":"university.dank.void","color":"light_purple","hoverEvent":{"action":"show_text","contents":[{"translate":"university.dank.voidtext1"},{"translate":"university.dank.voidtext2","color":"red"},{"translate":"university.dank.voidtext3"}]}}]

**Comment by desht** [comment]
Yeah sorry I never replied to this one originally.  This is actually a lot more difficult than it might initially appear, and while I agree it would be very to cool to have, I still haven't quite figured out how feasible it would be to implement without massive internal changes to how quest text is managed...

**Comment by desht** [comment]
Since we have multi-page quests now for handling very large descriptions, is this one needed anymore?

**Comment by aaronhowser1** [comment]
Not really
